model/model.wl

```diff
@@ -253,87 +252,86 @@ getStateParams[state_]:=Module[{raw,pop,dist,buckets},
 stateParams = Association[{#->getStateParams[#]}&/@Keys[fitStartingOverrides]];
 
 
-(* Define the model to be evaulated in the simulations -- 
-all parameters are given either from the Monte-Carlo draws or evaluting the means *)
-(* This defines an enriched SEIR model with extra states for things like PCR confirmation and fatalities *)
-(* It also fires several events for things like when the hospital / icu capacities are exceeded so that
-those don't have to be computed manually later *)
-
-(* functions in the model are:
-Sq - the currently susceptible population
-
-Eq - the currently infected population
-
-Iq - the currently infectious population
-ISq - infectious and no care needed
-IHq - infectious and in hospitalized non-ciritcal condition
-ICq - infectious and in critical condition
-
-PCR - cumulative PCR confirmations
-
-RSq - recovered without needing care
-
-RepHq - hospitalized patient reported in data
-EHq - cumulative hospitalized patients
-HHq - hospitalized patients (actual)
-RHq - recovered from hospital without critical care
-
-HCq - hospitalized and in need of critical care
-CCq - in ciritcal care
-RCq - recovered from critical care
-Deaq - passed after critical care
-
-est - initial infection impulse (eg from imported cases at importtime)
-*)
+(* Define the model to be evaulated in the simulations -- all parameters are given either from the Monte-Carlo draws or evaluting the means *)
+(* This defines an enriched SEIR model with extra states (e.g., hospitalized, deceased) and reporting (e.g., cumulative PCR rates) *)
+(* It also fires several events for things like when the hospital / icu capacities are exceeded so that those don't have to be computed manually later *)
 
 testTraceExposedDelay0=10;
 
 (* a set of parameters take from California that can be used for code testing purposes *)
 generateModelComponents[distancing_] := <|
   "equationsODE" -> Flatten[{
+      (**** Enriched SEIR model section ****)
```

Comment:
I reordered some and refactored these equations for clarity, grouping them by eSEIR equations versus the cumulative and reporting functions.